ReLU

migeed-z opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #61310
* #61265
* #61239
* #61238
* #61188
* **#61150**

Differential Revision: [D29625826](https://our.internmc.facebook.com/intern/diff/D29625826)

migeed-z committed:
ReLU

[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

eellison commented on test/fx/test_gradual_type.py:
```diff
@@ -406,5 +406,59 @@ def forward(self, x: TensorType((5, 2, 3, 4))):
+        for n in traced.graph.nodes:
+            if n.target == 'output':
+                assert isinstance(n.type, TensorType)
+                assert torch.Size(n.type.__args__) == B.forward(torch.rand(2, 2, 4, 5)).size()
```
you should know that resnet output is fixed at [1, 1000] so checking the output doesn't really check the inference well

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z committed:
Update on "ReLU"




[ghstack-poisoned]

migeed-z commented:
@migeed-z has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29625826).

jamesr66a reviewed (approved):
![image](https://user-images.githubusercontent.com/4685384/125005992-eff7ea80-e011-11eb-848b-d583999c7c32.png)

migeed-z committed:
Update on "ReLU"


Differential Revision: [D29625826](https://our.internmc.facebook.com/intern/diff/D29625826)

[ghstack-poisoned]

migeed-z commented:
@migeed-z has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29625826).

migeed-z committed:
Update on "ReLU"


Differential Revision: [D29625826](https://our.internmc.facebook.com/intern/diff/D29625826)

[ghstack-poisoned]

migeed-z commented:
@migeed-z has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29625826).

migeed-z commented:
@migeed-z has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29625826).
